src/data/dataMethods.js

```diff
@@ -14,12 +14,12 @@ const excludeFalsey = (param, defValue) => {
 
 const getMutationQuery = (options = {}) => {
   return {
+    dryRun: options.dryRun,
     returnIds: true,
     returnDocuments: excludeFalsey(options.returnDocuments, true),
     visibility: options.visibility || 'sync',
-    ...(options.skipCrossDatasetReferenceValidation && {
-      skipCrossDatasetReferenceValidation: options.skipCrossDatasetReferenceValidation,
```

Comment:
I see this was removed here previously. Would it make sense to remove the undefined ones still (if they end up being part of the query - in order to compress URLs)?

Reply:
It automatically drops `undefined` values, while boolean parameters are set to `true`/`false` - so I think the change makes it easier to read.